onos_ric_sdk_py/sdl.py

```diff
@@ -74,6 +75,96 @@ async def get_cell_ids(self, e2_node_id: str) -> List[str]:
         except GRPCError as e:
             raise ClientRuntimeError() from e
 
+    async def _get_cell_entity_id(self, e2_node_id: str, cell_id: str) -> int:
+        """
+        given e2_node_id and cell_id, returns entity id
+        raises ClientRuntimeError if cell_id is not found
+        """
+        if not self._ready:
+            raise ClientStoppedError()
+
+        client = TopoStub(self._topo_channel)
+        filters = Filters(
+            relation_filter=RelationFilter(
+                src_id=e2_node_id,
+                relation_kind=RanRelationKinds.CONTAINS.name.lower(),
+                target_kind="",
+            )
+        )
+
+        try:
+            response = await client.list(filters=filters)
+            for obj in response.objects:
+                if obj.entity.kind_id != RanEntityKinds.E2CELL.name.lower():
+                    continue
+
+                aspects = obj.aspects["onos.topo.E2Cell"].value
+                e2_cell = E2Cell().from_json(aspects)
+                if e2_cell.cell_object_id == cell_id:
+                    return obj.id
+        except GRPCError as e:
+            raise ClientRuntimeError() from e
+
+        raise ClientRuntimeError(
+            f"cannot find cell_id:{cell_id} in e2_node:{e2_node_id}"
+        )
```

Comment:
I think it would be more appropriate for this function to return `None` in this case.

Reply:
I couldn't decide between returning None and doing what python does - raise KeyError or IndexError. I'll change this